File: src/Entity/Question.php
<?php

namespace App\Entity;
use Symfony\Component\Validator\Constraints as Assert;

use App\Repository\QuestionRepository;
use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity(repositoryClass=QuestionRepository::class)
 */
class Question
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")

     */



    private $id;

    /**
     * @ORM\Column(type="text")
     */

    /**
     * @ORM\Column(type="text", length=255)
     * @Assert\Length(
     *      min = 10,
     *      max = 200,
     *      minMessage = "L'enonce doit comporter au moins {{ limit }} caractères",
     *      maxMessage = "L'enonce doit comporter au plus {{ limit }} caractères"
     * )
     */
    private $enonce;


    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\Length(
     *      min = 3,
     *      max = 255,
     *      minMessage = "La reponse doit comporter au moins {{ limit }} caractères",
     *      maxMessage = "la reponse doit comporter au plus {{ limit }} caractères"
     * )
     */
    private $choix1;


    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\Length(
     *      min = 3,
     *      max = 255,
     *      minMessage = "La reponse doit comporter au moins {{ limit }} caractères",
     *      maxMessage = "la reponse doit comporter au plus {{ limit }} caractères"
     * )
     */
    private $choix2;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\Length(
     *      min = 0,
     *      max = 255,
     *      minMessage = "La reponse doit comporter au moins {{ limit }} caractères",
     *      maxMessage = "la reponse doit comporter au plus {{ limit }} caractères"
     * )
     */
    private $choix3;



    /**
     * @ORM\ManyToOne(targetEntity=Quiz::class, inversedBy="questions")
     * @ORM\JoinColumn(nullable=false)
     */
    private $quiz;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $reponse;



    public function getId(): ?int
    {
        return $this->id;
    }

    /**
     * @param mixed $id
     */
    public function setId($id): void
    {
        $this->id = $id;
    }

    public function getEnonce(): ?string
    {
        return $this->enonce;
    }

    /**
     * @return mixed
     */
    public function getChoix1()
    {
        return $this->choix1;
    }

    /**
     * @param mixed $choix1
     */
    public function setChoix1($choix1): void
    {
        $this->choix1 = $choix1;
    }

    /**
     * @return mixed
     */
    public function getChoix2()
    {
        return $this->choix2;
    }

    /**
     * @param mixed $choix2
     */
    public function setChoix2($choix2): void
    {
        $this->choix2 = $choix2;
    }

    /**
     * @return mixed
     */
    public function getChoix3()
    {
        return $this->choix3;
    }

    /**
     * @param mixed $choix3
     */
    public function setChoix3($choix3): void
    {
        $this->choix3 = $choix3;
    }






    public function setEnonce(string $enonce): self
    {
        $this->enonce = $enonce;

        return $this;
    }



    public function getQuiz(): ?Quiz
    {
        return $this->quiz;
    }

    public function setQuiz(?Quiz $quiz): self
    {
        $this->quiz = $quiz;

        return $this;
    }

    public function getReponse(): ?string
    {
        return $this->reponse;
    }

    public function setReponse(string $reponse): self
    {
        $this->reponse = $reponse;

        return $this;
    }



}

File: src/Models/Question.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Models;

/**
 *
 * @author Eya
 */
public class Question {
    private int id ;
     private int quiz_id ;
     private String enonce;
     private String choix1 ;
      private String choix2 ;
      private String choix3 ;
       private String reponse ;
       

    public Question(int id, int quiz_id ,String enonce, String choix1, String choix2, String choix3,  String reponse) {
        this.id = id;
         this.quiz_id = quiz_id;
        this.enonce = enonce;
       
        this.choix1 = choix1;
        this.choix2 = choix2;
        this.choix3 = choix3;
        this.reponse = reponse;
    }

    public int getId() {
        return id;
    }

  

    public String getEnonce() {
        return enonce;
    }

    public void setEnonce(String enonce) {
        this.enonce = enonce;
    }

    public String getChoix1() {
        return choix1;
    }

    public void setChoix1(String choix1) {
        this.choix1 = choix1;
    }

    public String getChoix2() {
        return choix2;
    }

    public void setChoix2(String choix2) {
        this.choix2 = choix2;
    }

    public String getChoix3() {
        return choix3;
    }

    public void setChoix3(String choix3) {
        this.choix3 = choix3;
    }

    public int getQuiz_id() {
        return quiz_id;
    }

    public void setQuiz_id(int quiz_id) {
        this.quiz_id = quiz_id;
    }

    public String getReponse() {
        return reponse;
    }

    public void setReponse(String reponse) {
        this.reponse = reponse;
    }

  
       
       
    
}
